rust/operator-binary/build.rs

```diff
@@ -1,3 +1,8 @@
+use stackable_commons_crd::authentication::AuthenticationClass;
+use stackable_operator::crd::CustomResourceExt;
+
 fn main() {
     built::write_built_file().expect("Failed to acquire build-time information");
+
+    AuthenticationClass::write_yaml_schema("../../deploy/crd/authentication.crd.yaml").unwrap();
```

Comment:
I think we are about to retire this from the operators, do we still need this here, or can we just leave it out?

Reply:
Thx